spec/request/target/create_spec.rb

```diff
@@ -71,5 +70,24 @@
         end
       end
     end
+
+    context 'when limit reached' do
+      before do
+        10.times do
+          user.targets.create!(attributes_for(:target, topic_id: topic.id))
+        end
+      end
```

Comment:
Maybe here you can use the `create_list` from `FactoryBot` that lets you create multiple records ([create_list](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#building-or-creating-multiple-records)).
You could do something like this:
```ruby
before do
  create_list(:target, 10, user: user, topic: topic)
  subject
end
```
And with this you can also avoid the `subject` in each test.